src/MyMarketManager.WebApp/Program.cs

```diff
@@ -24,6 +24,24 @@
 builder.Services.AddScoped<DbContextMigrator>();
 builder.Services.AddHostedService<DatabaseMigrationService>();
 
+// Add Azure AI Vision service for image vectorization
+var visionEndpoint = builder.Configuration["AzureVision:Endpoint"] ?? "";
+var visionApiKey = builder.Configuration["AzureVision:ApiKey"] ?? "";
```

Comment:
These should come from the aspire app host.

I see we are missing the fact that aspire needs to provision the AI foundry resources for image and text embeddings and image analysis.

Look up the latest docs on what should be done.